### ISSUE TITLE
aktualizr dbus issue

### ISSUE THREAD

**Issue by rojarfast1991** [advancedtelematic/aktualizr#802]
Hi Team,

We are building AGL 5.0.2 code base for sota feature,where you have used sota server which is taking with aktualizr(sota client) for m3ulcb renesas hardware.

But in our case aktualizr is not able to capture Dbus event.

Please suggest us, is there any way to capture the dbus event in  @aktualizr.

**Comment by patrickvacek** [comment]
Once upon a time, we had limited dbus support for a feature that ended up getting cut, and since then, we removed the dbus code since it wasn't being maintained and probably didn't work. We do, however, still maintain a socket gateway, although it hasn't gotten much use recently. If this is something you want to have, we always welcome PRs, or we can discuss your options to meet your needs.

(cc: @cajun-rat @OYTIS)
